.github/workflows/kfp-kubernetes-test-python38.yaml

```diff
@@ -0,0 +1,28 @@
+name: kfp kubernetes tests with Python 3.8
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-kubernetes-test-python38.yaml'
+      - 'sdk/python/**'
+      - 'api/v2alpha1/**'
+      - 'kubernetes_platform/**'
+
+jobs:
+  kfp-kubernetes-test-python38:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: Run tests
+        # original file is using sudo
+        run: ./test/platform/kfp-kubernetes-execution-tests.sh
```

Comment:
Looks like you're calling the wrong file here. This is the file that needs to run as part of this test: https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml#L229

Reply:
Good catch!!!